backend/app/crud/user.py

```
@@ -46,42 +52,44 @@ def authenticate_user(*, db: Session, email: str, password: str) -> User | None:
+    db.commit()
+    db.refresh(prt)
+    return prt
+
+
+def get_valid_password_reset_token(
+    *, db: Session, token: str
+) -> Optional[PasswordResetToken]:
+    prt = db.exec(
+        select(PasswordResetToken).where(
+            PasswordResetToken.token == token,
+            PasswordResetToken.used == False,
```

Ruff (0.11.9)

Avoid equality comparisons to `False`; use `if not PasswordResetToken.used:` for false checks

Replace with `not PasswordResetToken.used`

(E712)